Module '"../../std/io/bufio"' has no exported member 'EOF'.

Error trace:
```
error TS2305: Module '"../../std/io/bufio"' has no exported member 'EOF'.

► https://deno.land/x/dejs/dejs.ts:4:21

4 import { BufReader, EOF } from 'https://deno.land/std/io/bufio.ts';
```

Environment:
```
❯❯❯ deno version
deno: 0.11.0
v8: 7.7.37
typescript: 3.5.1
```

Maybe pinning the std version could fix this? I see other modules like alosaur worked around this by only importing from std@0.9.0

--- a/dejs.ts
+++ b/dejs.ts
@@ -1,7 +1,7 @@
 const { open, Buffer } = Deno;
-import Reader = Deno.Reader;
+type Reader = Deno.Reader;
 import { stringsReader } from 'https://deno.land/std/io/util.ts';
-import { BufReader, EOF } from 'https://deno.land/std/io/bufio.ts';
+import { BufReader } from 'https://deno.land/std/io/bufio.ts';
 import escape from 'https://deno.land/x/lodash/escape.js';
 
 const globalEval = eval;
@@ -96,7 +96,7 @@ export async function compile(reader: Reader): Promise<Template> {
 
   while (true) {
     const byte = await src.readByte();
-    if (byte === EOF) {
+    if (byte === Deno.EOF) {
       break;
     }
 

